cli/flox/src/commands/show.rs

```diff
@@ -37,6 +38,10 @@ impl Show {
     pub async fn handle(self, flox: Flox) -> Result<()> {
```

Comment:
> `#[instrument(name = "show", fields(show_all = self.all`

Does this give us metrics in Sentry about real-world usage? Should it be removed at the same time?

Reply:
if we remove the flag we should also remove this here.
considering that the flag was basically defunct until now anyway, i dont see much of a point to keep metrics about it in case someone does use it.

Reply:
I was wondering if the metrics would tell us if/when it's safe to remove rather than hide the flag. Have we done that previously? Does it matter in this case?